apps/entropy-tester/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@pythnetwork/entropy-tester",
-  "version": "1.2.0",
+  "version": "1.3.0",
```

Comment:
Although this one does need to be versioned since this isn't a vercel app so please retain this change